packages/@expo/cli/src/serve/serveAsync.ts

```diff
@@ -66,6 +66,7 @@ async function startStaticServerAsync(dist: string, options: Options) {
     send(req, filePath, {
       root: dist,
       index: 'index.html',
+      extensions: ['html'],
```

Comment:
Interesting! This must've worked at some point I guess 🙃 cc @krystofwoldrich: Do we think this is something we should absorb into the `@expo/server` adapters? Usually, `@expo/server`'s scope is only the "server" output, but maybe it's worth increasing its scope just a tiny bit for the adapters? wdyt?

Reply:
Yeah, I think enhancing the adapters with a default behavior for static serving would make sense.

Although now `@expo/server` only handles dynamic serve now, in our docs examples we show how to setup static serving (some vendors like Vercel pickup the assets automatically).

If `@expo/server` adapter would include static serve, it would simplify the boiler plate users need to copy (at least initially).  

https://docs.expo.dev/router/reference/api-routes/#bun